uhdm-tests/opentitan/Makefile.in

```diff
@@ -244,14 +244,55 @@ aes_sbox.sv \
 aes_shift_rows.sv \
 aes_sbox_dom.sv \
 aes_mix_columns.sv \
-usbdpi.sv'
+usbdpi.sv \
+entropy_src_ack_sm.sv \
+entropy_src_main_sm.sv \
+entropy_src_core.sv \
+entropy_src_reg_top.sv \
+entropy_src.sv \
+tlul_pkg.sv \
+prim_lfsr.sv \
+prim_clock_gating.sv \
+prim_flop.sv \
+prim_generic_flop.sv \
+prim_flop_2sync.sv \
+prim_generic_flop_2sync.sv \
+prim_packer_fifo.sv \
+prim_alert_sender.sv \
+tlul_adapter_reg.sv \
+prim_subreg.sv \
+prim_fifo_sync.sv \
+prim_alert_receiver.sv \
+prim_esc_sender.sv \
+alert_handler_reg_top.sv \
+alert_handler_reg_wrap.sv \
+alert_handler.sv \
+alert_handler_ping_timer.sv'
 
+# Working set of Opentitan files
 TO_SURELOG = \
 	$(shell \
 		echo \'${SURELOG_FILE_LIST}\' | sed 's/\s/\\|/g' )
 
-# Need to readd this to .vc file due to double matching 'ibex_pkg.sv'
-READD='../src/lowrisc_ip_rv_core_ibex_pkg_0.1/rtl/rv_core_ibex_pkg.sv'
+# Need to readd parametrized modules for the SV frontend to use
+READD=../src/lowrisc_prim_generic_clock_gating_0/rtl/prim_generic_clock_gating.sv \
```

Comment:
As discussed, please change this to a burn-down list for the frontends to avoid listing files twice.